### PR TITLE
Add integrations page

### PR DESCRIPTION
**Pull request by signalnerve**
This is the docs write-up for using `wrangler-action` in various types of workflows. I've added it in a new "Integrations" section, but open to changing that. Here's where it lives now:

```  
Tooling
  |-> Wrangler
  |-> Integrations (NEW)
```

### CODE REVIEW

File: content/tooling/integrations/github.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+# Deploy your Workers application with GitHub Actions
+
+If you'd like to automate deploying your Workers application, we've published a GitHub Action to simplify the process. GitHub Actions allows you to build continuous integration workflows for your GitHub projects, and with [`wrangler-action`](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler), you can customize your workflow to deploy your project in situations like:
+
+- When a new commit is merged to your `master` branch
+- On a `cron`-like schedule, such as every hour, or daily
+- When an API "dispatches" an event to your GitHub repo
+
+First, we'll explore the basics of using the action, and explain how to configure your project to use it. After the basics, we'll cover each of the above deployment situations in detail, with a brief explanation and code example to help you get started. If you'd like to see the action itself, you can [find it in GitHub's marketplace](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler).
```
Comment (victoriabernard92) [inline content/tooling/integrations/github.md]:
```suggestion
First, we'll explore the basics of using the action, and explain how to configure your project to use it. After the basics, we'll cover each of the above deployment situations in detail with a brief explanation and code example to help you get started. If you'd like to see the action itself, you can [find it in GitHub's marketplace](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler).
```
I would link to Github actions docs in this top section here

File: content/tooling/integrations/github.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      - name: Publish
+        uses: signalnerve/wrangler-action@0.1.4
+        with:
+          apiKey: ${{ secrets.apiKey }}
+          email: ${{ secrets.email }}
+```
+
+## Configuration
+
+You'll need to configure Wrangler using GitHub's Secrets feature - go to "Settings -> Secrets" and add your Cloudflare API key and email (for help finding these, see the [Workers documentation](https://developers.cloudflare.com/workers/quickstart/#finding-your-cloudflare-api-keys)). Your API key and email are encrypted by GitHub, and the action won't print them into logs, so they should be safe!
```
Comment (victoriabernard92) [inline content/tooling/integrations/github.md]: We don't want to reiterate this information or maintain the Github actions instructions, so I suggest we just point to github's article and the finding api key article
```suggestion
You'll need to configure the API keys for Wrangler using GitHub's Secrets feature. First, [grab your Cloudflare API Key](https://developers.cloudflare.com/workers/quickstart/#finding-your-cloudflare-api-keys) then [add it to Github Secrets](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables). Your API key and email are encrypted by GitHub and the action won't print them into logs, so they should be safe!
```

**Comment by signalnerve** [comment]
hey @victoriabernard92 – I would like to keep the other examples! I definitely understand your point that it's duplicating something from the Actions docs, but I think there's a use in giving people code samples that they can drop directly into their projects. the alternative is that they need to go through the GitHub Actions docs, learn how the syntax works, and then apply it in their own workflows. 

by keeping the samples here, users interested in working with Actions can, ideally, get all the info they need to start working with the action, directly from our docs. my head is always in the bounce rate space – how can we keep people in _our_ docs, getting all the relevant info they need, and not just send them immediately to someone else's documentation? 

coming from that angle, I'd argue that we should be presenting people with a couple options of what they can do, even if it duplicates some info from the GitHub docs, because at the end of the day, they're still just trying to deploy a workers application: having all the info they need to do that, and understanding the options that this integration gives them seems valuable, without them having to be sent to dig through another set of docs.

**Comment by victoriabernard92** [comment]
I see your point. I like making things easy for users to use our product also, but this article is focussed on github's product. 

Whether we keep it or not I think should be determined by the end objective. I was under the impression we are targetting _users already using Github actions_, and this is an easy step to use wrangler support but the objective is not to teach them Gitub actions which get much more complicated than what is in this article.  If our objective is to target _users not already using Github and teach them Github actions_, then though I disagree with the objective's importance,  I do think you are right we should not only keep all 3 examples but also add more examples and maybe have a Github section.

Also keep in mind Github actions are still in beta, so should we really be banking on the functionality not to change?

**Comment by victoriabernard92** [comment]
Thought about this why not just include this info in the action itself? 

I suggest:
1. Structure: 
  ```
  Tooling
    |-> Wrangler
    |-> Integrations #single page 
  ```
2. Integration be a single page that lists links to integrations (i.e. for now just list a link to [wrangler's github action](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler) )
3. Maintain all github actions docs, in one place: [wrangler's github action](https://github.com/marketplace/actions/deploy-to-cloudflare-workers-with-wrangler)

**Comment by signalnerve** [comment]
sure, i can work with that! will update

File: content/tooling/integrations.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: 'Integrations'
+new: true
+---
+
+We're currently working on building integrations for Cloudflare Workers with other tools, especially if they make continuous integration and continuous deployment easier. Below is a list of the current integrations officially supported by Cloudflare Workers:
```
Comment (victoriabernard92) [inline content/tooling/integrations.md]:
```suggestion
We're currently working on building integrations for Cloudflare Workers to optimize your continuous integration and continuous deployment flow. The integrations officially supported for Cloudflare Workers are:
```

**Pull request merged**